test/e2e/send.feature

```diff
@@ -8,19 +8,14 @@ Feature: Send dialog
     And I click "send button"
     And I wait 1 seconds
     Then I should see text "Transaction is being processed and will be confirmed. It may take up to 15 minutes to be secured in the blockchain." in "result box message" element
-
-  @advanced
-  Scenario: should allow to send with second passphrase
-    Given I'm logged in as "second passphrase account"
-    And I fill in "1" to "amount" field
-    And I fill in "94495548317450502L" to "recipient" field
-    And I click "send next button"
-    And I fill in second passphrase of "second passphrase account" to "second passphrase" field
-    And I click "second passphrase next"
-    When I click "send button"
-    And I wait 1 seconds
-    Then I should see text "Transaction is being processed and will be confirmed. It may take up to 15 minutes to be secured in the blockchain." in "result box message" element
-
+    Then I click "dashboard" menu
+    And I should see 3 rows
+    When I click "seeAllLink"
+    Then I should be on url "/main/transactions"
+    And I should see 26 rows
+    When I scroll to the bottom of "transaction results"
+    Then I should see 51 rows
```

Comment:
Adding these 7 lines to `test/e2e/send.feature` doesn't make much sense. I know you want to have bigger scenarios, but it's making it harder to find what is covered by e2e tests and what is not.